common/pubkey.go

```diff
@@ -31,6 +32,18 @@ var EmptyPubKey PubKey
 
 // NewPubKey create a new instance of PubKey
 // key is bech32 encoded string
+
+func GetAddressFromPubkeyString(pubkey string) (sdk.AccAddress, error) {
```

Comment:
Maybe we could add unit test for this method

Reply:
This method is not needed, I removed it.